crypto/fipsmodule/rand/urandom.c

```diff
@@ -276,11 +274,14 @@ static void wait_for_entropy(void) {
     return;
   }
 
-#if defined(BORINGSSL_FIPS) && !defined(URANDOM_BLOCKS_FOR_ENTROPY)
```

Comment:
Are there any calls at the beginning of wait_for_entropy() that need to be bypassed for MacOS or iOS? Also is the problem only in FIPS builds?